elearning-frontend/src/components/actions/saveWord.js

```diff
@@ -0,0 +1,22 @@
+import Toastify from "../../core/Toastify"
+import apiClient from "../../services/api"
+
+const saveWord = (data, clearFields) => {
+    apiClient({
+        method: 'post',
+        url: "/api/admin/quizzes",
+        data: {
+            category_id: data.category_id,
+            word: data.word,
+            choices: data.choices,
+        }
+    }).then(response => {
```

Comment:
```suggestion
    apiClient({
        method: 'post',
        url: "/api/admin/quizzes",
        data,
    }).then(response => {
```

you can do this instead